usersession/agent/rest_api.go

```diff
@@ -244,7 +258,7 @@ func postPendingRefreshNotification(c *Command, r *http.Request) Response {
 	}
 
 	// TODO: this message needs to be crafted better as it's the only thing guaranteed to be delivered.
-	summary := fmt.Sprintf(i18n.G("Pending update of %q snap"), refreshInfo.InstanceName)
+	summary := fmt.Sprintf(i18n.G("Pending update of “%s” snap"), refreshInfo.InstanceName)
```

Comment:
what is the reason  for this change? The quotes look odd

Reply:
They are "typographic quotes". I was requested to do that change first, but now I see that I have another ask to change it to "normal quotes"... (or maybe I misunderstood it...)

Reply:
I'll change them.